test/discretization.jl

```diff
@@ -1,4 +1,13 @@
 @testset "Discretization" begin
+  function is_fully_connected(mesh::UnstructuredMesh)
+    inds = collect(1:length(vertices(mesh)))
```

Comment:
```suggestion
    inds = collect(1:nelements(mesh))
```

Reply:
The number of elements is the number of triangles. Here, we really want the number of vertices. However that makes me think that the number of triangles will exactly be `n-2`, `n` being the number of points, so I'll add this test.